app/assets/locales/android_translatable_strings.txt

```diff
@@ -868,7 +868,8 @@ settings.main.title=CommCare > Settings
 settings.advanced.title=CommCare > Advanced
 clear.user.data=Clear User Data
 clear.user.data.warning.title=WARNING!
-clear.user.data.warning.message=You are about to delete all unsent and incomplete forms. This is a PERMANENT action and you will NOT be able to recover these deleted forms. Make sure this is what you want to happen before continuing.
+clear.user.data.warning.message.generic=Are you sure you want to clear all mobile user data? Once you clear user data, you will need to sync with the server to get all your data again.

```

Comment:
we can keep the key probably same here as `clear.user.data.warning.message` as the earlier message was also very generic and keeping it same won't need anyone to add a new translation in case they were overriding the translation for this key.